IanGRap/Squish
# Animations pass 2

Contributors: Ian, Wesley

Goal: Implement the second pass of skeligoo animations in unity

Important Files: 
- Skeligoo Controller.controller: The new controller for the new animations and mesh.
- Player.prefab: Added the new mesh with animations. Removed animation coordinator. Made the bone mesh white instead of green.
- PlayerDash.cs, PlayerMovement.cs, PlayerJump.cs, PlayerGrounded.cs: Update the animator directly.

How to test:
- Load the AnimationTest scene and move around as skeligoo. Animations should play for your inputs
- Load up sprint 4 and the animations should still play and look pretty good.

Notes:
- The skull sometimes slips out of the goo during the dash but its pretty minor. I think this is just worth waiting on more iterations on the animations.
- The down dash animation is just the up dash animation. The transitions were super wonky for the down dash. I thought skeligoo pieroeting down was kinda cool though.
